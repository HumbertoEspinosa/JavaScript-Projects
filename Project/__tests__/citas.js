import Citas from '../js/classes/Citas'

describe('Probar la clase de citas',()=>{
    const citas = new Citas();
    const id=Date.now();
    test('Agregar nueva cita',() =>{
        const citaObj = {
            id,
            mascota: 'Any',
            propietario: 'Martha',
            telefono: '132132132',
            fecha: '10-12-2022',
            hora:'9:30',
            sintomas: 'unknown'
        }

       
        citas.agregarCita(citaObj);
        expect(citas).toMatchSnapshot();
        
    })

    test('Actualizar cita',()=>{
        const citaObj = {
            id,
            mascota: 'Any update',
            propietario: 'Martha',
            telefono: '132132132',
            fecha: '10-12-2022',
            hora:'9:30',
            sintomas: 'unknown update'
        }
        citas.editarCita(citaObj);
        expect(citas).toMatchSnapshot();
    })

    test ('Eliminar cita',()=>{
        citas.eliminarCita(id)
        expect(citas).toMatchSnapshot();
    })
})