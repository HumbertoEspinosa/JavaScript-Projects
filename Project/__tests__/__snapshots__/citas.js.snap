// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probar la clase de citas Actualizar cita 1`] = `
Citas {
  "citas": Array [
    Object {
      "fecha": "10-12-2022",
      "hora": "9:30",
      "id": 1659981832606,
      "mascota": "Any update",
      "propietario": "Martha",
      "sintomas": "unknown update",
      "telefono": "132132132",
    },
  ],
}
`;

exports[`Probar la clase de citas Agregar nueva cita 1`] = `
Citas {
  "citas": Array [
    Object {
      "fecha": "10-12-2022",
      "hora": "9:30",
      "id": 1659981832606,
      "mascota": "Any",
      "propietario": "Martha",
      "sintomas": "unknown",
      "telefono": "132132132",
    },
  ],
}
`;

exports[`Probar la clase de citas Eliminar cita 1`] = `
Citas {
  "citas": Array [],
}
`;
